docs/book/introduction.md

```diff
@@ -4,27 +4,178 @@ description: Welcome to ZenML!
 
 # ⭐ Introduction
 
-🤹 Are you an ML engineer or data scientist shipping models to production while juggling a plethora of tools?
+ZenML is an extensible, open-source MLOps framework for creating portable, 
+production-ready machine learning pipelines.
+By decoupling infrastructure from code, ZenML gives all developers in your 
+organization more freedom and independence in how they approach their work.
 
-🤷‍♂️ Do you struggle with versioning data, code, and models in your projects?
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
 
-👀 Have you had trouble replicating production pipelines and monitoring models in production?
+{% tabs %}
 
-[🙇](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Are you dealing with lots of unstructured notebooks or scripts with lots of manual processes?
+{% tab title="ZenML for Data Scientists" %}
 
-[🚧](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Do you find it challenging to transition your ML workflows from the experimentation setting to a production-ready environment?
+As a Data Scientist, ZenML gives you the freedom to focus on modeling and
+experimentation while writing code that is immediately production-ready.
 
-If you answered yes to any of the questions above, ZenML is here to help you:
+**Develop Locally:** Since ZenML code runs in any environment, you can 
+develop ML models locally with all your favorite tools. Once you are happy with 
```

Comment:
Non-sequitur: if you can run your code in any environment, shouldn't the second part of this sentence be about how you can run locally AND in the cloud? (I know you get to that in the next sentence, but it makes this first sentence read funny).

Reply:
True, how would you formulate it then?

Reply:
```
**Develop Locally:** ZenML allows you to develop ML models in any environment using your favorite tools. When you're satisfied with your results locally, for example, a single command facilitates your transition to a production environment, where all your previously developed code will continue to function effectively.
```

Maybe?

Reply:
Great suggestion, thanks